src/main/java/ru/sbt/mipt/oop/EventHandlingLoop.java

```diff
@@ -0,0 +1,86 @@
+package ru.sbt.mipt.oop;
+
+import static ru.sbt.mipt.oop.SensorEventType.*;
+
+public class EventHandlingLoop {
+
+    public static void handleEvents(SmartHome smartHome) {
+        // начинаем цикл обработки событий
+        while (true) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%BD%D0%B5%D1%81%D0%B8%D1%82%D0%B5-%D0%BB%D0%BE%D0%B3%D0%B8%D0%BA%D1%83-%D1%86%D0%B8%D0%BA%D0%BB%D0%B0-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B8-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%B2-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D0%BA%D0%BB%D0%B0%D1%81%D1%81